features/conversation/conversation-preview/conversation-preview-messages.query.ts

```diff
@@ -0,0 +1,33 @@
+import { conversationPreviewMessagesQueryKey } from "@/queries/QueryKeys";
+import {
+  ConversationMessagesQueryData,
+  conversationMessagesQueryFn,
+} from "@/queries/use-conversation-messages-query";
+import logger from "@/utils/logger";
+import { useQuery, UseQueryOptions } from "@tanstack/react-query";
+import { ConversationTopic } from "@xmtp/react-native-sdk";
+
+export const useConversationPreviewMessages = (
+  account: string,
+  topic: ConversationTopic,
+  options?: Partial<UseQueryOptions<ConversationMessagesQueryData>>
+) => {
+  return useQuery({
```

Comment:
I think as best practice we always extgract into const getConversationPreviewMessageQueryOPtions = ...